### ISSUE TITLE
Multiple Script Firing

### ISSUE THREAD

**Issue by RoamingMargins** [Tampermonkey/tampermonkey#749]
## Expected Behavior
I've fought this for a while, and reduced it down to the bare min, and it feels like it is a bug of some sort. 

https://www.amazon.com/s?i=merchant-items&me=A3SED09YUL6I5H&marketplaceID=ATVPDKIKX0DER
Is an example link of where this is happening at.

Essentially I'm attempting to write a Tampermonkey script to load the pagination results for me. Done it 100 times, should be simple, you'd think.

When I load, it fires off three times, no matter what I do. I can't capture it. I've added the noframes flag since it felt like the frames were causing it (not actual frames, loaded with srcdoc), I've tried adding a boolean variable to catch it, but all three instances are completely unaware of each other. So I'm at a complete loss.

## Actual Behavior

Script loads multiple times. Appends the same result 3 times.

## Specifications

- Chromium: (i.e. 75.0.3770.142 (Official Build))
- TM: (i.e. 4.8.41)
- OS: (i.e. Windows 10)

## Script

(Please give an example of the script if applicable.)
Script is below, here is my "test link", but any merch link does it.

// ==UserScript==
// @name         Test Issue
// @namespace    http://tampermonkey.net/
// @noframes
// @version      0.1
// @description  try to take over the world!
// @author       You
// @match   *://www.amazon.com/s?i=merchant-items*
// @require http://ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js
// @grant   GM_addStyle
// @grant   GM_setValue
// @grant   GM_getValue
// @grant   GM_xmlhttpRequest
// @grant   GM_log
// @run-at document-idle
// ==/UserScript==

let current_body = $('.s-result-list');
let count = 1;
function how_many_times() {
    unsafeWindow.console.warn("once", count, window.top == window.self);
    if(count < 2){
      unsafeWindow.console.log("Count is", count);
    }
    count++;
    current_body.append('<p><a href="#" id="tester">' + ' ' + count + ' ' + (window.top == window.self) + '</a></p>');
}
how_many_times();
```

**Comment by lokinder1** [comment]
why it is closed, I also facing the same issue
